aws_lambda_powertools/utilities/parameters/ssm.py

```diff
@@ -188,6 +191,72 @@ def _get(self, name: str, decrypt: bool = False, **sdk_options) -> str:
 
         return self.client.get_parameter(**sdk_options)["Parameter"]["Value"]
 
+    def set(
```

Comment:
set is a built-in name in Python and we should avoid that.

Reply:
Changed it over to `_set`